pkg/apiserver/api_installer.go

```diff
@@ -148,7 +148,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		if err != nil {
 			return nil, err
 		}
-		parentMapping, err := a.group.Mapper.RESTMapping(parentKind, a.group.Version)
+		parentMapping, err := a.group.Mapper.RESTMapping(parentKind, a.group.GroupVersion.String())
```

Comment:
> Do you want to change the signature of RESTMapping to use GroupVersion instead of string?

I've just started by adjusting `RESTMapping` to use a `GroupVersionKind` and rippling that out instead of `APIVersion` and `Kind`